trackmania.js

```diff
@@ -611,11 +632,13 @@ export async function leaderboard(live_service, oauth_service, track_info, lengt
         });
     }
 
+    const { groupUid, timestamp } = toBase64(track_info.groupUid, track_info.endTimestamp);
+
     let buttons = [{
                 type: MessageComponentTypes.BUTTON,
                 style: ButtonStyleTypes.SECONDARY,
                 label: 'First',
-                custom_id: `lb_f;${track_info.groupUid};${track_info.mapUid};${length}`,
+                custom_id: `lb${timestamp}_f;${groupUid};${track_info.mapUid};${length}`,
```

Comment:
nit: `toBase64` shouldn't add an underscore prefix, it hurt my pea brain when I saw this cus I thought this would generate a string like `lb12312381` instead of `lb_12312382`. Instead, this string should be smth like `lb_${timestamp}...` for clarity. Close this one if you want cus I might just be dumb